test/ExampleEnd2EndTest.php

```diff
@@ -274,13 +274,15 @@ private function runExample($testFile)
     {
         $this->testFile = $testFile;
         $examplesDir = realpath(__DIR__ . '/../examples');
-        $samplesTestCase = $examplesDir . '/' . $testFile;
+        $samplesTestCase = $examplesDir . DIRECTORY_SEPARATOR . $testFile;
         $logFile = tempnam(sys_get_temp_dir(), 'phpunit_log_');
         $environmentVariables = [];
         foreach ($this->environment as $name => $value) {
-            $environmentVariables[] .= "$name=$value";
+            $var = "$name=$value";
+            $environmentVariables[] = DIRECTORY_SEPARATOR==='\\' ? "set $var && " : $var;
```

Comment:
is this a portability problem? Doesn't seem to work for me on Linux and bash:
```
$ a=42 bash -c 'echo $a'
42
$ set a=42 && bash -c 'echo $a'

```

Reply:
windows doesn't handle setting  environment variables the same in it's command line, so this just makes sure the test doesn't break on windows.

Reply:
Would be interesting to later see if there is a way to cross-test with Windows on Travis CI, I don't think it is provided as a platform